src/solver.cpp

```diff
@@ -700,6 +701,8 @@ inline SolverRegistrar::SolverRegistrar(IdRegistryData& registry)
              Primitive::MultiMarginLoss,
              multimarginloss::MultiMarginLossForward{}.SolverDbId());
 
+    Register(registry, ++id, Primitive::Indexing, gather::GatherV2Backward{}.SolverDbId());
+
```

Comment:
Dont add a space between the newly added register and the comment below [[ref]](https://github.com/ROCm/MIOpen/blob/a3c5a57f145029672e285d82378208ca87edff0d/src/solver.cpp#L712C5-L713C70)

Reply:
updated it.